app.json

```
@@ -0,0 +1,24 @@
+{
+    "name": "vali-ah-backend",
+    "description": "1kbIdeas: The one-stop shop for all the self-help ideas you need",
+    "scripts": {},
```

Please include a postdeploy script to help set up the database after each deploy. e.g 
```suggestion
    "scripts": {
      "postdeploy": "npm run db:migrate"
    },
```